src/main/steps/upload-documents/template.njk

```diff
@@ -63,48 +63,64 @@
   text: uploadRequirements | safe
 }) }}
 
-         {% if uploadedDocuments.length > 0 %}
-    <ul class="govuk-list">
+{% if uploadedDocuments.length > 0 %}
+  <ul class="govuk-list">
   {% for document in uploadedDocuments %}
-      <li class="uploadedFile govuk-!-padding-top-2 govuk-!-padding-bottom-3 govuk-section-break govuk-section-break--visible">
-        <table class="documentRelevanceAndDeletion">
-          <tr>
-            <td class="documentRelevanceAndDeletionAlignment">{{ document.fileName }}
-            {% if document.description %}
-              <br><br>
-            {{ document.description }}</td>
-            {%endif%}
-            {% set documentId = document.documentId %}
-            <td class="documentRelevanceAndDeletionAlignment"><a href="{{postURL}}?removeId={{documentId}}" class="govuk-link--no-visited-state float-right">{{ delete }}</a></td>
-          </tr>
-        </table>
-      </li>
-    {%endfor%}
+    <li class="uploadedFile govuk-!-padding-top-2 govuk-!-padding-bottom-3 govuk-section-break govuk-section-break--visible">
+      <table class="documentRelevanceAndDeletion">
+        <tr>
+          <td class="documentRelevanceAndDeletionAlignment">{{ document.fileName }}
+          {% if document.description %}
+            <br><br>
+          {{ document.description }}</td>
+          {%endif%}
+          {% set documentId = document.documentId %}
+          <td class="documentRelevanceAndDeletionAlignment"><a href="{{postURL}}?removeId={{documentId}}" class="govuk-link--no-visited-state float-right">{{ delete }}</a></td>
+        </tr>
+      </table>
+    </li>
+  {%endfor%}
```

Comment:
Small thing; can we add spaces between the % and endfor

Reply:
Still need to turn {%endfor%} to {% endfor %} in the line above